src/store/bankdata/getters.ts

```diff
@@ -1,21 +1,39 @@
 import { GetterTree } from 'vuex';
 import { BankAccount } from '@src/view_models/BankAccount';
 import { Validity } from '@src/view_models/Validity';
+import { looksLikeIban } from '@src/util/bank_account_number_helpers';
 
+// It's important to note here that !bankDataIsInvalid is not the same as bankDataIsValid and
+// vice versa. This is because we have states Validity.INCOMPLETE and Validity.RESTORED. If
+// the validity of one of the values is one of these states then both of these will return false.
 export const getters: GetterTree<BankAccount, any> = {
 	bankDataIsInvalid: function ( state: BankAccount ): boolean {
-		return state.validity.bankdata === Validity.INVALID;
+		if ( state.values.accountNumber === '' || looksLikeIban( state.values.accountNumber ) ) {
```

Comment:
I think I need some in-code comment/explanation for the usage of these getters that would explain why an empty account number is valid OR invalid (depending on which getter you call). I find that very confusing.

Reply:
I already added a comment about it in the getters file, you think that would be enough?

Reply:
Which getters file? If you insert a reference comment here that would suffice if there was *some* form of documentation explaining the logic behind these 2 store getters.